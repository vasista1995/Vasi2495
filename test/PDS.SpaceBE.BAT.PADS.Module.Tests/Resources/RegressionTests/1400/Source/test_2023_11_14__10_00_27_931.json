{
  "IdSource": "SPACEACT2:BAT:BE:RU307316B11:BE1I:Mould:MoldTemperature:29881:240187155:0",
  "Data_flat_Metadata": {
    "Device": "U5880B",
    "ParameterClass": "E_SPC",
    "CFComment": "EOL",
    "Segment": "MOLD",
    "FourDReport": "*",
    "ParameterName": "MoldTemperature",
    "ParameterUnit": "deg",
    "ParameterOper": "Mould",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "ChannelId": "29881",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-11-14T06:43:46Z",
    "CreatedTimestamp": "2023-11-14T13:43:46Z",
    "UpdatedTimestamp": "2023-11-14T13:43:46Z",
    "UpdatedTimestampUtc": "2023-11-14T06:43:46Z",
    "ExportedTimestampUtc": "2023-11-14T13:43:46Z",
    "ExportedTimestamp": "2023-11-14T13:43:46Z",
    "SampleTimestampUtc": "2023-11-14T06:43:46Z",
    "SampleTimestamp": "2023-11-14T13:43:46Z",
    "SourceDataLevel": "C",
    "ChannelName": "Mold_Temperature/170-180",
    "ChannelDescr": "Created from LDS \nexclude list by \naction 'Create SPC Channel'",
    "RvStoreFlag": "Y",
    "CalcCreated": "12/02/2020 13:21:58",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 1187337,
    "ViolCount": 0,
    "CalculationStrategy": "ND",
    "LdsID": "1400",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "Y",
        "SpecHigh": 180.0,
        "SpecLow": 170.0,
        "SpecTarget": 175.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYNNYNYYYYYYYYYYYNNNYYYYYYNNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "175,23",
        "Median": "175,2",
        "Sigma": ",176698110409316",
        "Min": "174,9",
        "Max": "175,5",
        "Q1": "175,125",
        "Q3": "175,375"
      },
      "ControlLimits": {
        "ctrl_mean_low": 172.0,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 175.0,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 178.0,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "Y",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "CtrlSigmaHighEnabled": "N",
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 2.5,
        "ctrl_range_high": 5.0,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 175.226041666667,
        "ExtEwmaMV": 175.224950639202,
        "ExtMsMV": 0.0172848324290022,
        "EwmaS": 0.209216587951253,
        "EwmaR": 0.712594518781769,
        "ExtEwmaMVLCL": 0.0,
        "ExtEwmaMVCenter": 0.0,
        "ExtEwmaMVUCL": 0.0,
        "ExtMaMVLCL": 0.0,
        "ExtMaMVCenter": 0.0,
        "ExtMaMVUCL": 0.0,
        "EwmaSLCL": 0.0,
        "EwmaSCenter": 0.0,
        "EwmaSUCL": 0.0,
        "EwmaRLCL": 0.0,
        "EwmaRCenter": 0.0,
        "EwmaRUCL": 0.0,
        "ExtMSMVLCL": 0.0,
        "ExtMSMVCenter": 0.0,
        "ExtMSMVUCL": 0.0,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 0.0,
        "RawUal": 0.0,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 240187155,
        "Seqnr": 6,
        "Value": 175.2,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240187155,
        "Seqnr": 7,
        "Value": 174.9,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240187155,
        "Seqnr": 8,
        "Value": 175.1,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240187155,
        "Seqnr": 9,
        "Value": 175.4,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240187155,
        "Seqnr": 10,
        "Value": 175.2,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240187155,
        "Seqnr": 1,
        "Value": 175.4,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240187155,
        "Seqnr": 2,
        "Value": 175.1,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240187155,
        "Seqnr": 3,
        "Value": 175.3,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240187155,
        "Seqnr": 4,
        "Value": 175.2,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240187155,
        "Seqnr": 5,
        "Value": 175.5,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 10
  },
  "ProductionAction": {
    "Id": "SPACEACT1:BAT:BE:BE1I:Mould:MoldTemperature:29881",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "BAT:BE:RU307316B11:BE1I:Mould:MoldTemperature:29881:240187155:0"
  }
}