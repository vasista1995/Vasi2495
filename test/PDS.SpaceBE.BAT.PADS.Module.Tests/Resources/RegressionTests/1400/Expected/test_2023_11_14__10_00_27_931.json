{
  "Data_flat_Metadata": {
    "ParameterOper": "Mould",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-11-14T06:43:46Z",
    "BeginTimestamp": "2023-11-14T13:43:46Z",
    "EndTimestampUtc": "2023-11-14T06:43:46Z",
    "EndTimestamp": "2023-11-14T13:43:46Z"
  },
  "Data_flat_StrangeMetadata": {
    "Device": "U5880B",
    "ParameterClass": "E_SPC",
    "Segment": "MOLD"
  },
  "Data_1list_Parameters": [
    {
      "Data_1list_RawValues": [
        {
          "Seqnr": 6,
          "SampleId": 240187155,
          "Value": 175.2,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:43:46Z",
          "SampleTimestampUtc": "2023-11-14T06:43:46Z",
          "CreatedTimestamp": "2023-11-14T13:43:46Z",
          "CreatedTimestampUtc": "2023-11-14T06:43:46Z",
          "UpdatedTimestamp": "2023-11-14T13:43:46Z",
          "UpdatedTimestampUtc": "2023-11-14T06:43:46Z",
          "NumViolations": 0,
          "SampleMean": 175.23,
          "SampleMin": 174.9,
          "SampleMax": 175.5,
          "SampleMedian": 175.2,
          "SampleStdev": 0.176698110409316,
          "SampleQ1": 175.125,
          "SampleQ3": 175.375,
          "SampleSize": 10
        },
        {
          "Seqnr": 7,
          "SampleId": 240187155,
          "Value": 174.9,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:43:46Z",
          "SampleTimestampUtc": "2023-11-14T06:43:46Z",
          "CreatedTimestamp": "2023-11-14T13:43:46Z",
          "CreatedTimestampUtc": "2023-11-14T06:43:46Z",
          "UpdatedTimestamp": "2023-11-14T13:43:46Z",
          "UpdatedTimestampUtc": "2023-11-14T06:43:46Z",
          "NumViolations": 0,
          "SampleMean": 175.23,
          "SampleMin": 174.9,
          "SampleMax": 175.5,
          "SampleMedian": 175.2,
          "SampleStdev": 0.176698110409316,
          "SampleQ1": 175.125,
          "SampleQ3": 175.375,
          "SampleSize": 10
        },
        {
          "Seqnr": 8,
          "SampleId": 240187155,
          "Value": 175.1,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:43:46Z",
          "SampleTimestampUtc": "2023-11-14T06:43:46Z",
          "CreatedTimestamp": "2023-11-14T13:43:46Z",
          "CreatedTimestampUtc": "2023-11-14T06:43:46Z",
          "UpdatedTimestamp": "2023-11-14T13:43:46Z",
          "UpdatedTimestampUtc": "2023-11-14T06:43:46Z",
          "NumViolations": 0,
          "SampleMean": 175.23,
          "SampleMin": 174.9,
          "SampleMax": 175.5,
          "SampleMedian": 175.2,
          "SampleStdev": 0.176698110409316,
          "SampleQ1": 175.125,
          "SampleQ3": 175.375,
          "SampleSize": 10
        },
        {
          "Seqnr": 9,
          "SampleId": 240187155,
          "Value": 175.4,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:43:46Z",
          "SampleTimestampUtc": "2023-11-14T06:43:46Z",
          "CreatedTimestamp": "2023-11-14T13:43:46Z",
          "CreatedTimestampUtc": "2023-11-14T06:43:46Z",
          "UpdatedTimestamp": "2023-11-14T13:43:46Z",
          "UpdatedTimestampUtc": "2023-11-14T06:43:46Z",
          "NumViolations": 0,
          "SampleMean": 175.23,
          "SampleMin": 174.9,
          "SampleMax": 175.5,
          "SampleMedian": 175.2,
          "SampleStdev": 0.176698110409316,
          "SampleQ1": 175.125,
          "SampleQ3": 175.375,
          "SampleSize": 10
        },
        {
          "Seqnr": 10,
          "SampleId": 240187155,
          "Value": 175.2,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:43:46Z",
          "SampleTimestampUtc": "2023-11-14T06:43:46Z",
          "CreatedTimestamp": "2023-11-14T13:43:46Z",
          "CreatedTimestampUtc": "2023-11-14T06:43:46Z",
          "UpdatedTimestamp": "2023-11-14T13:43:46Z",
          "UpdatedTimestampUtc": "2023-11-14T06:43:46Z",
          "NumViolations": 0,
          "SampleMean": 175.23,
          "SampleMin": 174.9,
          "SampleMax": 175.5,
          "SampleMedian": 175.2,
          "SampleStdev": 0.176698110409316,
          "SampleQ1": 175.125,
          "SampleQ3": 175.375,
          "SampleSize": 10
        },
        {
          "Seqnr": 1,
          "SampleId": 240187155,
          "Value": 175.4,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:43:46Z",
          "SampleTimestampUtc": "2023-11-14T06:43:46Z",
          "CreatedTimestamp": "2023-11-14T13:43:46Z",
          "CreatedTimestampUtc": "2023-11-14T06:43:46Z",
          "UpdatedTimestamp": "2023-11-14T13:43:46Z",
          "UpdatedTimestampUtc": "2023-11-14T06:43:46Z",
          "NumViolations": 0,
          "SampleMean": 175.23,
          "SampleMin": 174.9,
          "SampleMax": 175.5,
          "SampleMedian": 175.2,
          "SampleStdev": 0.176698110409316,
          "SampleQ1": 175.125,
          "SampleQ3": 175.375,
          "SampleSize": 10
        },
        {
          "Seqnr": 2,
          "SampleId": 240187155,
          "Value": 175.1,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:43:46Z",
          "SampleTimestampUtc": "2023-11-14T06:43:46Z",
          "CreatedTimestamp": "2023-11-14T13:43:46Z",
          "CreatedTimestampUtc": "2023-11-14T06:43:46Z",
          "UpdatedTimestamp": "2023-11-14T13:43:46Z",
          "UpdatedTimestampUtc": "2023-11-14T06:43:46Z",
          "NumViolations": 0,
          "SampleMean": 175.23,
          "SampleMin": 174.9,
          "SampleMax": 175.5,
          "SampleMedian": 175.2,
          "SampleStdev": 0.176698110409316,
          "SampleQ1": 175.125,
          "SampleQ3": 175.375,
          "SampleSize": 10
        },
        {
          "Seqnr": 3,
          "SampleId": 240187155,
          "Value": 175.3,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:43:46Z",
          "SampleTimestampUtc": "2023-11-14T06:43:46Z",
          "CreatedTimestamp": "2023-11-14T13:43:46Z",
          "CreatedTimestampUtc": "2023-11-14T06:43:46Z",
          "UpdatedTimestamp": "2023-11-14T13:43:46Z",
          "UpdatedTimestampUtc": "2023-11-14T06:43:46Z",
          "NumViolations": 0,
          "SampleMean": 175.23,
          "SampleMin": 174.9,
          "SampleMax": 175.5,
          "SampleMedian": 175.2,
          "SampleStdev": 0.176698110409316,
          "SampleQ1": 175.125,
          "SampleQ3": 175.375,
          "SampleSize": 10
        },
        {
          "Seqnr": 4,
          "SampleId": 240187155,
          "Value": 175.2,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:43:46Z",
          "SampleTimestampUtc": "2023-11-14T06:43:46Z",
          "CreatedTimestamp": "2023-11-14T13:43:46Z",
          "CreatedTimestampUtc": "2023-11-14T06:43:46Z",
          "UpdatedTimestamp": "2023-11-14T13:43:46Z",
          "UpdatedTimestampUtc": "2023-11-14T06:43:46Z",
          "NumViolations": 0,
          "SampleMean": 175.23,
          "SampleMin": 174.9,
          "SampleMax": 175.5,
          "SampleMedian": 175.2,
          "SampleStdev": 0.176698110409316,
          "SampleQ1": 175.125,
          "SampleQ3": 175.375,
          "SampleSize": 10
        },
        {
          "Seqnr": 5,
          "SampleId": 240187155,
          "Value": 175.5,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:43:46Z",
          "SampleTimestampUtc": "2023-11-14T06:43:46Z",
          "CreatedTimestamp": "2023-11-14T13:43:46Z",
          "CreatedTimestampUtc": "2023-11-14T06:43:46Z",
          "UpdatedTimestamp": "2023-11-14T13:43:46Z",
          "UpdatedTimestampUtc": "2023-11-14T06:43:46Z",
          "NumViolations": 0,
          "SampleMean": 175.23,
          "SampleMin": 174.9,
          "SampleMax": 175.5,
          "SampleMedian": 175.2,
          "SampleStdev": 0.176698110409316,
          "SampleQ1": 175.125,
          "SampleQ3": 175.375,
          "SampleSize": 10
        }
      ],
      "CFComment": "EOL",
      "FourDReport": "*",
      "Data_flat_Limits": {
        "CKC_ID": "0",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "Y",
          "SpecLowEnabled": "Y",
          "spec_high": 180.0,
          "spec_low": 170.0,
          "target": 175.0
        },
        "ControlLimits": {
          "ctrl_mean_low": 172.0,
          "CtrlMeanLowEnabled": "Y",
          "ctrl_mean_target": 175.0,
          "CtrlMeanTargetEnabled": "Y",
          "ctrl_mean_high": 178.0,
          "CtrlMeanHighEnabled": "Y",
          "CtrlLowEnabled": "N",
          "CtrlHighEnabled": "N",
          "CtrlStdevLowEnabled": "N",
          "CtrlSigmaCenterEnabled": "N",
          "CtrlRangeCenterEnabled": "Y",
          "CtrlCenterEnabled": "N",
          "CtrlStdevHighEnabled": "N",
          "CtrlRangeLowEnabled": "N",
          "ctrl_range_target": 2.5,
          "ctrl_range_high": 5.0,
          "CtrlRangeHighEnabled": "Y"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 10,
        "base_count": 10,
        "flagged_count": 0,
        "mean": 175.23,
        "median": 175.2,
        "stdev": 0.1676305461424041,
        "range": 0.5999999999999943,
        "min": 174.9,
        "max": 175.5,
        "q02": 174.936,
        "q05": 174.99,
        "q25": 175.125,
        "q75": 175.375,
        "q95": 175.45499999999998,
        "q98": 175.482,
        "NumViolations": 0,
        "Samples": "240187155"
      },
      "ParameterName": "MoldTemperature",
      "ParameterUnit": "deg",
      "ChannelId": "29881",
      "SourceDataLevel": "C",
      "channel_name": "Mold_Temperature 170-180",
      "channel_description": "Created from LDS  exclude list by  action 'Create SPC Channel'",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-11-14T06:43:46Z",
      "created_timestamp": "2023-11-14T13:43:46Z",
      "updated_timestamp_utc": "2023-11-14T06:43:46Z",
      "updated_timestamp": "2023-11-14T13:43:46Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-11-14T06:43:46Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED2:BAT:BE:BE1I:Mould:",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "UniqueSourceKey": "BAT:BE:RU307316B11:BE1I:Mould:MoldTemperature:29881:240187155:0",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-11-14T13:43:46Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "Measlot::SPACEAGGED2:BAT:BE:BE1I:Mould::ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "BAT"
  }
}